Sloth.Core/Extensions/StringExtensions.cs

```diff
@@ -19,5 +21,38 @@ public static string Base64Decode(this string base64EncodedData)
             var base64EncodedBytes = System.Convert.FromBase64String(base64EncodedData);
             return System.Text.Encoding.UTF8.GetString(base64EncodedBytes);
         }
+
+        public static string SafeTruncate(this string value, int max)
+        {
+            if (string.IsNullOrWhiteSpace(value) || value.Length <= max)
+            {
+                return value;
+            }
+
+            if (max <= 0)
```

Comment:
@jSylvestre I'm thinking if this is really "safe" we should set max to 0 if the value is <0.  so that there is no way the Safe* method can fail